docs/node/quickstart.mdx

```diff
@@ -12,6 +12,10 @@ slug: /node/quickstart
 To run either a single Kwil node, or a network of nodes, you will need to download the binaries.  Kwil releases
 binaries on [Github](<https://github.com/kwilteam/binary-releases/releases>).
 
+Kwil also requires a dedicate PostgreSQL host configured for `kwild`.
+
+See the Installation (link!) page for details.
+
 ## Single Node
 
```

Comment:
Should we also add instructions for postgres quickstart in these docs?

I'm not sure what would be the best way to do that... Maybe provide the docker compose file in a `<details></details>` component? Or say: "go to `deployments/compose/postgres` in the kwil-db repo and run `docker compose up -d`"? Thoughts?

Reply:
Related: https://github.com/kwilteam/kwil-db/pull/557

We should be able to do `docker run` with that image pushed to our hub account.

Reply:
With https://github.com/kwilteam/kwil-db/pull/557, they can now do docker run without a compose file.  We'd just need to push that image to our hub account.

But if we want to go one step further, we could do the `curl`+`docker compose` thing mentioned in https://github.com/kwilteam/docs/pull/82#discussion_r1504728660 to start kwild and postgres containers in one go.  This would mean they don't even run the kwild binary directly in the Quickstart, so maybe that's not what we want.  I think the tutorials and the other guides should use the binaries directly, but perhaps we can be open to the quickstart being a single command just to get an RPC service running so they can learn how to use Kwil DB on a local random (and disposable!) chain.